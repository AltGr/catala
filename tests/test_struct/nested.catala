@Article@

/*
new struct S:
  data x type S 

new scope A:
  param y type S

scope A:
  def y := S { -- x: 1 }
*/