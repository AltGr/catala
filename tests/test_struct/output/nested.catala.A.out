catala: too many arguments, don't know what to do with `test_struct/bad/nested2.catala', `test_struct/bad/nested.catala'
Usage: catala [OPTION]... BACKEND FILE
Try `catala --help' for more information.
